scripts/Muon/GUI/Common/contexts/muon_context.py

```diff
@@ -131,9 +133,50 @@ def _do_rebin(self):
                (self.gui_context['RebinType'] == 'Variable' and
                 'RebinVariable' in self.gui_context and self.gui_context['RebinVariable'])
 
+    def get_workspace_names_for_FFT_analysis(self, use_raw=True):
```

Comment:
should this be in the ADS handler as it is naming?

Reply:
I've updated this so that the name of the PhaseQuad workspace is created by a function in workspace_naming module. I think this function as a whole is ok here though as it is just pulling together a list of names to populate the input workspace combo box with and the generation of the names is all done by functions from the ADS handler module.

Reply:
ok